tests/integration/targets/test_mysql_db/tasks/state_present_absent.yml

```diff
@@ -18,8 +18,8 @@
 # ============================================================
 - name: remove database if it exists
   command: >
-    "{{ mysql_command }} -sse 'drop database {{ db_name }}'"
-  ignore_errors: True
+    "{{ mysql_command }} -sse 'DROP DATABASE IF EXISTS {{ db_name }}'"
+  ignore_errors: true
 
```

Comment:
Is this related to the issue ? 🤨

Reply:
No sorry. I did that when the tests were failing on python 3.6.

Should I revert that?

Reply:
@laurent-indermuehle 

> No sorry. I did that when the tests were failing on python 3.6.

I'm curious why they fail if they are not related to the changes. If it's safe, we can leave it as is.